VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -623,6 +623,7 @@ private void showNetworkError() {
                         }
 
                         isNetworkMessageDisplayed = true;
+                        isShownWifiSettingsMessage = false;
```

Comment:
it is not a good practice to add different flag to each message. 
should we wrap them into message event or something?